docs/content/en/docs/getting-started/optional/api-server-extra-args.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "API Server Extra Args"
+linkTitle: "API Server Extra Args"
+weight: 60
+description: >
+  EKS Anywhere cluster yaml specification API Server Extra Args reference
+---
+
+## API Server Extra Args support (optional)
+
+#### Provider support details
+|                | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:--------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+| **Supported?** |   ✓	    |     ✓      |   	 ✓   |     ✓      |  ✓   |
+
+In order to configure a cluster with API Server extra args, you need to configure your cluster by updating the cluster configuration file to include the details below. The feature flag `API_SERVER_EXTRA_ARGS_ENABLED=true` needs to be set.
+
+This is a generic template with some example API Server extra args configuration below for reference:
+```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+   name: my-cluster-name
+spec:
+    ...
+    controlPlaneConfiguration:
+        apiServerExtraArgs:
```

Comment:
Can these be updated for existing clusters on all of the providers? (ie. by running an `eksctl anywhere upgrade` and by applying new spec with K8s API-compatible tooling)

Reply:
Yes